math/pronicnumber.go

```diff
@@ -0,0 +1,24 @@
+// pronicnumber.go
+// description: Returns true if the number is pronic, false otherwise
+// details: A number n is a pronic number if
+// it is equal to product of two consecutive numbers m and m+1.
+// wikipedia: https://en.wikipedia.org/wiki/Pronic_number
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see pronicnumber_test.go
+
+package math
+
+import "math"
+
+// This function returns true if argument passed to the function is pronic, false otherwise.
```

Comment:
Please use Go's documenting conventions.

```suggestion
// PronicNumber returns true if argument passed to the function is pronic and false otherwise.
```

Also, I think you should provide a definition of what `pronic` means.